### ISSUE TITLE
Ongoing call produces notifications within Element

### ISSUE THREAD

**Issue by aWeinzierl** [vector-im/element-android#4513]
### Steps to reproduce

1. get called (various scenarios: Element is closed or open or you are even in the corresponding chat)
2. accept the call 
3. leave the chat/call window (don't exit the actual call)
4. view the list of chats which contains the chat where that call was placed
5. see (multiple) notifications despite no new messages (but probably call events in the background - nothing the user wouldn't be already aware of);
![image](https://user-images.githubusercontent.com/20560137/142451131-ccdf1428-0e18-4260-89e1-9176594d3e0e.png)

#### What did you expect?

- no new notifications 
- no highlighting

In other words: the entry in the room list should look like any other room entry pointing to a room in which nothing new has happened (aside perhaps an indicator that the ongoing call is in the specific room)

#### What happened instead?

Indication of 5 new ~~messages~~ events which I already know of and won't find when I open the chat, because there are no new real messages.

### Your phone model

Fairphone 3+

### Operating system version

Android 10

### Application version and app store

Element version 1.3.8 [40103082] (G-b4480)

### Homeserver

matrix.org

### Will you send logs?

No
